python/GangaLHCb/Lib/Splitters/GaussSplitter.py

```diff
@@ -77,7 +77,11 @@ def split(self, job):
             first = self.firstEventNumber + i * self.eventsPerJob + 1
             opts = 'from Gaudi.Configuration import * \n'
```

Comment:
It is needed to import ```ApplicationMgr``` - I guess I could move this to be specifically under the Gauss application.